### ISSUE TITLE
FEEDBACK: GNSS (3) - your form, our feature service, your webhook

### ISSUE THREAD

**Issue by marikavertzonis** [tedrick/Survey123-Testing#140]
This scenario allows you to **create a survey** in your org that points to our **existing feature service**. It will also allow you to explore the concept of **webhooks by creating one of your own**.

1) take this survey and publish it in your own org.
    - drag and drop the spreadsheet  [Street Lights.xlsx](https://github.com/tedrick/Survey123-Testing/files/2330382/Street.Lights.xlsx) on the new survey tile of Connect
    - copy this image to the media folder
![streetlights](https://user-images.githubusercontent.com/5713624/44759954-01d99380-ab80-11e8-99a4-fe13580cd9f5.png)
    - publish your survey

2) Follow the steps here to set up a webhook in Microsoft Flow. http://doc.arcgis.com/en/survey123/browser/analyze-results/webhooks.htm#ESRI_SECTION1_C52B77E37FAD462AB58F09982E381240
No further configuration in the survey123.arcgis.com is required when using Flow. If you have another webhook provider - follow the steps at the beginning of the help topic.

3) Sign in to Survey123 on your device and download this new survey that you published

4) connect a GNSS receiver to your device. Follow steps on EAC https://earlyadopter.esri.com/project/article/item.html?cap=e69ef91f45744b98882c651f7b518eb7&arttypeid={5a8e51ed-4a87-43fe-a67d-2b7829af130f}&artid={9AEF4173-09CC-4134-9811-0AA2B3B405DD}

5) go outside and collect the 3 points described in the survey

6) send the records from the field (if you have data connection) or send upon return to the office
    - see the notification emails sent to wherever you set them up to go
    - take a look at the points collected in the feature service here http://www.arcgis.com/home/item.html?id=0921493eec9844b4affcf02f1e9c2200

**Comment by JohnHasthorpe** [comment]
- Britany
- Chris
- Gareth
- Paul

**Comment by paul-gpsit** [comment]
Followed the steps and got this working.
Only comment (to help with documentation) was that, after creating the 'Flow', I wrongly assumed that I needed to copy the "payload url" thing and go back and create the webhook manually in Survey123 web app. I didn't realise that the connector automatically added the webhook.

**Comment by paul-gpsit** [comment]
Re connecting to external GNSS device, I connected to one of the Bad Elf gps devices. I had issues with sign in (due to network connection) meaning I had to force close the app. When I reopened the app, I went straight into the survey but the gps location wasn't available and I wasn't sure why. After a minute or so I thought to go back to the Settings page and check the location source setting. The 'Bad Elf' connection had a status of 'connecting' then connected a few seconds later and everything worked fine from there.
I don't know if it was coincidental that it connected a few seconds after going to that page or whether going to that page triggered something for it to connect.
